.coveragerc

```diff
@@ -10,6 +10,6 @@ omit =
      */imaginary/pyparsing.py
 
 [paths]
-source =
+source_paths =
```

Comment:
As I read it, this name doesn't matter.  I guess you have a reason for this change?  Alignment with other coverage configs you have?

https://coverage.readthedocs.io/en/coverage-5.3/config.html#paths

Reply:
Yea not a meaningful change.  There was an intermediate version where I had multiple entries in `[paths]` to try to get the coverage merged properly but it was bogus.  Gonna back out this change since it didn't lead anywhere.